routes/api.php

```diff
@@ -29,4 +29,15 @@
         Route::put('/apps/{id}', 'update');
         Route::delete('/apps/{id}', 'destroy');
     });
+
+    /**
+     * Companies Routes.
+     */
+    Route::group(['controller' => '\Kanvas\Http\Controllers\Companies\CompaniesController'], function () {
```

Comment:
@rwhite27 isn't it better to use CompaniesController::class  ?

Reply:
maybe. I'll leave the full namespace reference though.